.pre-commit-config.yaml

```diff
@@ -78,7 +78,7 @@ repos:
         language: python
         entry: ./tools/hooks/sync_dependencies.py
         files: pyproject.toml
-        additional_dependencies: ["mne"]
+        additional_dependencies: ["mne==1.9.0"]
```

Comment:
Couldn't figure out why the Python min req wasn't updating locally but it was on Azure. It's because the env here wasn't updating. I think this is just one more thing we'll have to update at release time